### PR TITLE
Module 1 Week 1: Hello Microverse Project

### PR DESCRIPTION
**Pull request by the-catalystmc**
Hello Microverse Project Week 1

In this project, the following were done:

- Added the .gitignore and linter files.
- Added the index.html and style.css files.
- Added a ReadMe file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+Twitter: @twitterhandle
+LinkedIn: LinkedIn
+
+Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a star if you like this project!
```
Comment (Davidosky007) [inline README.md]: Your readme file should be descriptive as stated in the project requirement 👍 this makes your repository look professional 💯

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @the-catalystmc, 

👍 Good work on your commit messages, its descriptive keep it up 👍 
 
Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I suggest your readme file be more descriptive as stated in the project requirement
* your pull request should have a descriptive title telling what changes are inside and also it could have a short summary describing introduced changes in more details. these make reviewing easy.
* you should consider fixing the linter check error as it makes your code clean and bug-free. 👍 

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

# Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
+Authors
+GitHub: @the-catalyst
+
+Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a star if you like this project!
```
Comment (Temmarie) [inline README.md]: - [ ] As required by the previous reviewer, your readme should be descriptive and ordered. Kindly use [this](https://github.com/microverseinc/readme-template) template for your readme, customising it to suit your needs.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @the-catalystmc    ! :wave: :wave:

#### You have done really well in implementing the previously suggested requirements.
- You used proper GitHub Flow,
- Your linter checks are all passing,
- Your  PR message is descriptive. 👍🏾 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Recommended** ♻️

- [ ] Kindly use the provided template below to help organise your readme, making it clearer and more professional.


_See the inline comments below for more required changes_

- - - 
**Note that you still have 1 more review as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

File: README.md (hunk 2 above)
Comment (the-catalystmc) [reply README.md]: Is it the readme overall or the specified line? I have followed the template and customized it as necessary, am I expected to include all the unnecessary headings in the template? Or should I have more information on the specified line?

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @the-catalystmc! 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

that's how you do it


Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**